src/Control/Carrier/Accum/Strict.hs

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE FlexibleInstances #-}
```

Comment:
Is this module necessary, given that we already have `AccumT` as carrier?

Reply:
I prefer it if only because we get to have handler functions (e.g. `runReader`) with their parameters in a more ergonomic order. I've been considering whether we need to worry about the carrier type as well, but I haven't come to any decision yet. For the time being, let's go with it.